src/entry_points/cli.py

```diff
@@ -213,21 +211,6 @@ def consume_all(pytest_args, help_flag, pytest_help_flag):
     pytest.main(args)
 
 
-@click.command(context_settings=dict(ignore_unknown_options=True))
-@common_options
-def fill_and_consume_all(pytest_args, help_flag, pytest_help_flag):
-    """
-    Fill and consume test fixtures using all available consume commands.
-    """
-    args = handle_help_flags(pytest_args, help_flag, pytest_help_flag)
-
-    temp_dir = Path(tempfile.TemporaryDirectory().name) / "fixtures"
-    args += ["--output", temp_dir]
-    pytest.main(args)
-    consume_args = get_hive_flags_from_env()
-    pytest.main(["-c", "pytest-consume-all.ini", "--input", temp_dir, "-v"] + consume_args)
```

Comment:
I don't know why I removed this: should I add this back @danceratopz?

Reply:
Haha, saw this yesterday - I was going to ask you! :smile:

Reply:
I'll add it back :D